SourceControl.md

```diff
@@ -0,0 +1,54 @@
+# General
+
+In the Triple .NET team we use Git for source control management. How you we [Git](https://git-scm.com) - whether it is via a GUI or the CLI - is up to us. Our commits however must stick to the guidelines:
+
+1. We must commit one change per commit
+2. We must adhere to the commit title and -description requirements
+3. We should not commit generated files
+4. We should not commit personal user settings
+5. We must include the .editorconfig in the root of your solution
+
+## Commit title and description
+
+We should strive to make our commits concise and descriptive, allowing to understand what happens within by only reading its title. By starting our commits with the following keywords we communicate our intent of the change in a uniform and succinct manner. In the optional description, you can find space to freely describe your commit in more detail.
```

Comment:
Are these keywords from a standard somewhere? If so, reference it, if not, why these keywords specifically?

Reply:
These keywords I've picked from online articles about best practices for committing. There's more available, and I've seen other keywords being used in the team, so let's open up the discussion!